domains/oslc-domains/pom.xml

```diff
@@ -16,20 +16,45 @@
     -->
     <groupId>org.eclipse.lyo</groupId>
     <artifactId>oslc-domains</artifactId>
-    <version>4.2.0-SNAPSHOT</version>
+    <version>4.0.0</version>

```

Comment:
All the POM changes need to be rolled back.

Reply:
Can you clarify? why do we need to roll back?